src/stories/icons.stories.mdx

```diff
@@ -0,0 +1,155 @@
+import { IconItem,IconGallery,Meta,Story,Preview } from '@storybook/addon-docs/blocks';
+
+<Meta title="Icons"/>
+
+# Vocabulary Icons
+
+## Chevron
+<Preview>
+    <form>
```

Comment:
No need to wrap `<IconGallery>` in `<form>` and `<Preview>`.

Reply:
I wrapped it in Preview so that sample code is visible.

Reply:
When I tried removing Preview tag error pops up.